pkg/e2e/operators/managedupgrade.go

```diff
@@ -0,0 +1,304 @@
+package operators
+
+import (
+	"context"
+	"fmt"
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "github.com/openshift/api/config/v1"
+	upgradev1alpha1 "github.com/openshift/managed-upgrade-operator/pkg/apis/upgrade/v1alpha1"
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/prometheus"
+	prometheusv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"time"
+)
+
+var managedUpgradeOperatorTestName string = "[Suite: operators] [OSD] Managed Upgrade Operator"
```

Comment:
This should be put in `[Suite: informing]` since it's a new test.

Reply:
Thanks - done!